Week3/homework/step2-3.js

```diff
@@ -32,7 +38,12 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   let result = '';
 
   // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  // console.log(str, num, result);
+
+  do {
+    result += str;
+    num--;
+  } while (num > 0);
```

Comment:
👍 for logic but you could make it better by adding an if condition where you check if `num < 0` then return empty string otherwise use your logic 🙂